deepchem/models/graph_models.py

```diff
@@ -715,7 +715,7 @@ class _GraphConvKerasModel(tf.keras.Model):
   def __init__(self,
```

Comment:
Could you add a unit test for `GraphConvModel` that makes use of the dense layers?